src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -69,31 +72,25 @@ dependencies:
 Next, run `pub get packages`, then import the `flutter_localizations` library and specify
 `localizationsDelegates` and `supportedLocales` for `MaterialApp`:
 
-<!-- skip -->
+<?code-excerpt "gen_l10n_example/lib/main.dart (LocalizationDelegatesImport)"?>
 ```dart
 import 'package:flutter_localizations/flutter_localizations.dart';
-// TODO: uncomment the line below after codegen
-// import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+```
 
-// ...
+<?code-excerpt "gen_l10n_example/lib/main.dart (LocalizationDelegates)" replace="/App.*//g"?>
+```dart
+return Material
+  title: 'Localizations Sample
```

Comment:
Looks like the title is cut off as well: 
```suggestion
    title: 'Localizations Sample App',
```

Reply:
fixed now on the latest commit